site/content/posts/2021-11-09-E2E-Skeleton-Plugin.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Skip the Boilerplate: A Plugin Ready For Your Business Logic"
```

Comment:
@johnSchnake I think the title should say something about testing since the example (i.e. making testing easier, or running your test just got easier, or building tests with sonobuoy, etc).

Reply:
Also, maybe put e2e-skeleton (great name) in the title.

Reply:
You like the name? I've been on the fence about whether or not we needed a new one. My family is halloween-obsessed so I keep visualizing spooky skeletons :)